src/common/modals/AddInstance/InstanceName.js

```diff
@@ -77,17 +74,28 @@ const InstanceName = ({
 
   const imageURL = useMemo(() => {
     if (!modpack) return null;
-    // Curseforge
-    if (!modpack.synopsis) {
-      return modpack?.logo?.thumbnailUrl;
-    } else {
+    if (modpack.synopsis) {
       // FTB
       const image = modpack?.art?.reduce((prev, curr) => {
         if (!prev || curr.size < prev.size) return curr;
         return prev;
       });
       return image.url;
     }
+    if (Object.prototype.hasOwnProperty.call(modpack, 'solder')) {
```

Comment:
why did you do that instead of just modpack.hasOwnProperty('solder') or just modpack.solder?

Reply:
Sorry, I don't have that much experience with typescript, changed it now.